configs/watch_pending.cfg

```diff
@@ -40,7 +40,8 @@
         "^Ubuntu ASAN VM 12.04 x64.*": "tst-linux64",
         "^Ubuntu TSAN VM 12.04 x64.*": "tst-linux64",
         "^b2g_(emulator|ubuntu64)(-jb|-kk)?_vm(?!_large)": "tst-linux64",
-        "^Windows 7 VM-GFX 32-bit.*": "t-w732",
+        "^Windows 7 VM 32-bit.*": "t-w732",
```

Comment:
r- for this, you'll want:
   "Windows 7 VM(?!-GFX) 32-bit.*": "t-w732",

Otherwise the actually chosen slave could be wrong depending on the key ordering (both variants match)